src/register/vexriscv/mod.rs

```diff
@@ -0,0 +1,10 @@
+pub mod dci;
+pub use self::dci::*;

```

Comment:
`read_csr_as_usize` macro will declare a `read` function in each module. Is it useful to *-reexport all the module contents?

Reply:
No, I misunderstood which modules get re-exported.  It looks like only certain register groups -- namely those that end in `-x` -- get exported.  I'll remove the re-export.